### PR TITLE
Add in conditional for trioeval and triobin. Default is not run.

### PR DESCRIPTION
**Pull request by vsmalladi**

### CODE REVIEW

File: workflows/sample/tasks/hifiasm_trio.wdl [hunk 1]
```diff
@@ -103,6 +103,46 @@ task yak_trioeval {
...
+  Int memory = threads * 3              #forces at least 3GB RAM/core, even if user overwrites threads
+
+  command <<<
+    echo requested disk_size =  ~{disk_size}
+    echo
+    source ~/.bashrc
+    conda activate yak
+    echo "$(conda info)"
+
+    (yak triobin  -c1 -d1 -t ~{threads} -1 ~{parent1_yak} -2 ~{parent2_yak} ~{fasta_gz} > ~{yak_triobin_txt_name} ) > ~{log_name} 2>&1
```
Comment (williamrowell) [inline workflows/sample/tasks/hifiasm_trio.wdl]:
```
❯ yak triobin -h
Usage: yak triobin [options] <pat.yak> <mat.yak> <seq.fa>
Options:
  -c INT     min occurrence [2]
  -d INT     mid occurrence [5]
  -t INT     number of threads [8]
```

Comment (williamrowell) [reply workflows/sample/tasks/hifiasm_trio.wdl]: We need to remove the `-1` and `-2` from both yak calls.  This isn't part of the syntax.

```
> yak trioeval -h
Usage: yak trioeval [options] <pat.yak> <mat.yak> <seq.fa>
Options:
  -c INT     min occurrence [2]
  -d INT     mid occurrence [5]
  -n INT     min streak [2]
  -t INT     number of threads [8]

> yak triobin -h
Usage: yak triobin [options] <pat.yak> <mat.yak> <seq.fa>
Options:
  -c INT     min occurrence [2]
  -d INT     mid occurrence [5]
  -t INT     number of threads [8]
```

Comment (vsmalladi) [reply workflows/sample/tasks/hifiasm_trio.wdl]: Good catch. Removing.

**Comment by vsmalladi** [comment]
Ready to review @williamrowell

**Comment by williamrowell** [comment]
Looks like `yak trioeval` needs the update as well.  Neither uses the `-1/-2` syntax.

**Comment by williamrowell** [comment]
Thanks!  Let's give it a try.  @byoo, ready to test.

**Pull request merged**